yarn.lock

```diff
@@ -2833,7 +2833,7 @@
     google-protobuf "^3.17.0"
     long "^4.0.0"
 
-"@shapeshiftoss/types@^2.0.0", "@shapeshiftoss/types@^2.1.0":
+"@shapeshiftoss/types@^2.0.0":
```

Comment:
It seems like there is a yarn lock issue related to this types package version. We are trying to reference version ^3.0.0 from `market-service`, but there is only one version reference of `2.11.0` in the yarn lock. I would expect a v2.xx version for something like `unchained-client` that hasn't been updated along with a v3.xx version for `market-service`

Reply:
Disregard. Peer/Dev dependencies scenario...